src/App.jsx

```diff
@@ -6,6 +6,7 @@ import Header from "./components/Header";
 import { Switch, Route } from "react-router-dom";
 import auth from "./modules/auth";
 import { connect } from "react-redux";
+import PublishArticle from "./components/PublishArticle";
```

Comment:
Hmm... I wonder if something in line with "ArticleManagment" would be more appropriate as a component name?

Reply:
But then again... a lot of your components are about management...  🤔nah, forget it.